src/content/workshops/functions/1-what-is-serverless/_index.md

```diff
@@ -4,12 +4,109 @@ weight: 101
 outputs: ["Reveal"]
 ---
 
-# Slide 1
+## What is Serverless?
 
-Test
+- Execution model
+- Pay as you consume
+- No need to control scaling
+- Event Driven
 
+{{% note %}}
+Serverless is an execution model. Everytime you hear the word serverless, it's talking about this cloud focused execution model. Running the same code locally is a different concept.
+
+An attribute of that execution model is that you pay for what you consume and not for what you reserve. When you create services in cloud providers, you reserve for a certain amount of CPU/RAM. This is not a necessary concern when you start a new serverless application.
+
+Serverless goes without this as the whole premise to care about servers less. Meaning that when using a serverless framework, you should not be required to control scaling at all.
```

Comment:
```suggestion
The point of a serverless application is to focus on your application code, and less about the underlying infrastructure. 
```
Regarding this sentence:

> Meaning that when using a serverless framework, you should not be required to control scaling at all.

There are some relevant scaling concerns that customers may consider. Do you think there may be a better way to phrase this than "you should not be required to control scaling at all"?

Reply:
I love the way you phrased it.